storage_text_field/fields.py

```diff
@@ -76,6 +79,7 @@ def get_prep_value(self, value):
             )
         return super(StorageTextField, self).get_prep_value(file_path)
 
+    @lazy
```

Comment:
😢 
`Results are not memoized; the function is evaluated on every access.`

Reply:
but i dont think we would benefit from memoization (if that is a word).

Reply:
I'm fine with it not being memoized for the moment (it wasn't memozied before either). Memoization might help? But it's not what is causing the more immediate problem. I'm happy to add memoization in a follow up issue if we decide it needs to happen.